cardano-chain-gen/test/Test/Cardano/Db/Mock/Unit/Alonzo/Stake.hs

```diff
@@ -215,126 +216,128 @@ stakeDistGenesis :: IOManager -> [(Text, Text)] -> Assertion
 stakeDistGenesis =
   withFullConfigAndDropDB alonzoConfigDir testLabel $ \interpreter mockServer dbSync -> do
     startDBSync dbSync
-    a <- fillUntilNextEpoch interpreter mockServer
-    assertBlockNoBackoff dbSync (fromIntegral $ length a)
-    -- There are 5 delegations in genesis
-    assertEpochStake dbSync 5
+    blks <- fillUntilNextEpoch interpreter mockServer
+    assertBlockNoBackoff dbSync (fromIntegral $ length blks)
+    -- There are 10 delegations in genesis
```

Comment:
it can be removed as I think due to the previous implementation but the time the assertEpochStake was called the epoch 2 wouldn't have been inserted. This is why I changed the tests to check each epoch 1 + 2 separately